### PR TITLE
Fix a few minor warnings

### PR DESCRIPTION
**Pull request by abate**
lint related warning .

### CODE REVIEW

File: packages/extlib-compat/extlib-compat.1.6.1/opam [hunk 1]
```diff
@@ -1,13 +1,17 @@
-opam-version: "1"
+opam-version: "1.2"
 maintainer: "ygrek@autistici.org"
+author: "ygrek@autistici.org"
```
Comment (dsheets) [inline packages/extlib-compat/extlib-compat.1.6.1/opam]: This is typically a person's name.

Comment (abate) [inline packages/extlib-compat/extlib-compat.1.6.1/opam]: actually I really don't know what it his/her name ... ygrek ?

Comment (dsheets) [inline packages/extlib-compat/extlib-compat.1.6.1/opam]: Here is how ygrek credited itself a few days ago: https://github.com/ocaml/opam-repository/pull/4302/files

**Comment by abate** [comment]
ok, done.

**Comment by dsheets** [comment]
I think `ocamlfind` is also a `{build}` dependency (the constraint phrase only applies to the previous element, not all on the line). I've merged it now and if you'd like to make `ocamlfind` a build dep please open another PR.

**Pull request merged**